temporalcli/commands_test.go

```diff
@@ -411,13 +381,10 @@ func (d *DevServer) StartDevWorker(t *testing.T, options DevWorkerOptions) *DevW
 	w.Worker.RegisterActivity(ops.DevActivity)
 	// Start worker or fail
 	require.NoError(t, w.Worker.Start(), "failed starting worker")
+	t.Cleanup(w.Worker.Stop)
 	return w
 }
 
-func (d *DevWorker) Stop() {
```

Comment:
Same here, if I as the caller choose to start the worker, I should be able to choose to stop it